lib/column/tuple.go

```diff
@@ -566,6 +570,23 @@ func (col *Tuple) AppendRow(v any) error {
 		return nil
 	}
 
+	if tuple2, ok := v.(Tuple2); ok {
```

Comment:
Have you tried adding a case to switch above this for `reflect.Array`? Could work to have `v` be `[2]any`, tho I'm doubtful it'll ever be very good having to go through `reflect`